src/components/01-branding/02-typography/forms/data-entry-fields.njk

```diff
@@ -0,0 +1,32 @@
+<table class="usa-table">
```

Comment:
Restructure classes to use BEM  and just use in form components -- confusing here